dev-tools/build.sh

```diff
@@ -12,7 +12,7 @@ composer require --no-update symfony/polyfill-mbstring
 composer update --no-interaction --no-progress --no-dev --prefer-stable --optimize-autoloader
 composer info -D | sort
 
-composer show -d dev-tools humbug/box -q || composer update -d dev-tools --no-interaction --no-progress
+COMPOSER=composer_build.json composer show -d dev-tools humbug/box -q || composer update -d dev-tools --no-interaction --no-progress
```

Comment:
different composer files for dev-tools? we can as well just move to Phive

Reply:
And have back those ridiculous keyserver errors? This is (hopefully) only temporary (that's why this is still a draft - to show the update will work) until `humbug/php-scoper` will get release with `symfony/console` compatible with `^5`.